src/stores/ItemTemplateStore.js

```diff
@@ -23,7 +23,7 @@ class ItemTemplateStore {
       }));
   });
 
-  LoadItemTemplate = flow(function * ({itemTemplateId, force=false}) {
+  LoadItemTemplate = flow(async function * ({itemTemplateId, force=false}) {
```

Comment:
Flow functions are a special type of function used by asynchronous methods that update mobx state (i.e. observed attributes of the store. They are not `async` and they shouldn't have that keyword. Instead, wherever you would `await` in a flow function, you use `yield` instead.